lib/gui/pages/main/controllers/flash.js

```diff
@@ -111,9 +111,12 @@ module.exports = function(
       return 'Flash!';
     }
 
-    if (flashState.percentage === 0 && !flashState.speed) {
+    const PERCENTAGE_MINIMUM = 0;
```

Comment:
Does the same constant in multiple files imply that it out to be abstracted out?